xio-core/src/main/java/com/xjeffrose/xio/http/StreamingRequestData.java

```diff
@@ -62,6 +72,20 @@ public ByteBuf body() {
     return request.body();
   }
 
+  // endregion
```

Comment:
i'm not sure if this (pragma separating multiple interface implementations) is consistent with the project style